cpp/ql/src/semmle/code/cpp/rangeanalysis/RangeAnalysisUtils.qll

```diff
@@ -5,15 +5,27 @@ import cpp
  * relation) or 'non-strict' (a `<=` or `>=` relation).
  */
 newtype RelationStrictness =
+  /**
+   * Represents that a relation is 'strict' (that is, a `<` or `>` relation).
+   */
   Strict() or
+  /**
+   * Represents that a relation is 'non-strict' (that is, a `<+` or `>+` relation)
```

Comment:
```suggestion
   * Represents that a relation is 'non-strict' (that is, a `<=` or `>=` relation)
```
Or is this intentional? I might be missing some context.

Reply:
Unintentional.